### ISSUE TITLE
Swap Dark and Light Mode Buttons

### ISSUE THREAD

**Issue by rmfaheem** [quran/quran.com-frontend-v2#391]
## Issue
It seems that the buttons for the night and day themes do the reverse of what they should do.

### Illustration
Day mode:
![image](https://user-images.githubusercontent.com/51624008/116643392-059dc380-a982-11eb-9ca1-5f349707de67.png)

Night mode:
![image](https://user-images.githubusercontent.com/51624008/116643376-fcacf200-a981-11eb-8dd2-cd9cde5b12b3.png)

Minor issue but it should be reversed.

**Comment by Yuniac** [comment]
Assalamualaikum

I agree with you that it is a little bit inconvenient because the color is a little bit too transparent, does this look better to you?

Dark mode:

![image](https://user-images.githubusercontent.com/76709894/128597764-6ef3bd62-70bb-40a2-ab73-ffc4eda2effa.png)

Light mode:

![image](https://user-images.githubusercontent.com/76709894/128597767-a42606c5-2a80-4572-8fbf-71c1db14784c.png)

Or you meant that the currently active mode icon shouldn't have a modifier color but the inactive mode icon should have it? I think the idea here is the currently active mode has that greenish color to stand out and signals that it is the current active one.

**Comment by rmfaheem** [comment]
> I think the idea here is the currently active mode has that greenish color to stand out and signals that it is the current active one.

I agree that this is what we should be going for. But it does not seem to be case currently...?

**Currently**:
When dark mode is enabled, the light bulb is purple.
When light mode is enabled, the moon icon is purple.

**Should it not be**:
When **_dark mode_** is enabled, the **_moon icon is purple_**.
When **_light mode_** is enabled, the **_light bulb is purple_**.

**Comment by Yuniac** [comment]
I think not. I think as it is now is correct because when "When dark mode is enabled, the light bulb is purple." signals to the user that: "Hey, I'm a light pulp and I'm clickable" and the user tries and click so it swaps themes. And Allah knows best.

**Comment by Yuniac** [comment]
> 
> 
> Assalamualaikum
> 
> I agree with you that it is a little bit inconvenient because the color is a little bit too transparent, does this look better to you?
> 
> Dark mode:
> 
> ![image](https://user-images.githubusercontent.com/76709894/128597764-6ef3bd62-70bb-40a2-ab73-ffc4eda2effa.png)
> 
> Light mode:
> 
> ![image](https://user-images.githubusercontent.com/76709894/128597767-a42606c5-2a80-4572-8fbf-71c1db14784c.png)
> 
> Or you meant that the currently active mode icon shouldn't have a modifier color but the inactive mode icon should have it? I think the idea here is the currently active mode has that greenish color to stand out and signals that it is the current active one.

I can purpose this change in a PR too, btw. I personally think it fits better and more clear.

**Comment by Taimoorkhan1122** [comment]
> > Assalamualaikum
> > I agree with you that it is a little bit inconvenient because the color is a little bit too transparent, does this look better to you?
> > Dark mode:
> > ![image](https://user-images.githubusercontent.com/76709894/128597764-6ef3bd62-70bb-40a2-ab73-ffc4eda2effa.png)
> > Light mode:
> > ![image](https://user-images.githubusercontent.com/76709894/128597767-a42606c5-2a80-4572-8fbf-71c1db14784c.png)
> > Or you meant that the currently active mode icon shouldn't have a modifier color but the inactive mode icon should have it? I think the idea here is the currently active mode has that greenish color to stand out and signals that it is the current active one.
> 
> I can purpose this change in a PR too, btw. I personally think it fits better and more clear.

Assalam o alikum, I also noted this issues, its really confusing in terms of UX, I clicked light bulb for switching to light mode but it is disabled and currently highlighted 😕 if it's done intentionally (the highlight color) so, why the button which in not currently selected is disabled? It should toggle to light mode when I click the "light buld".

**Comment by Yuniac** [comment]
> 
> 
> > > Assalamualaikum
> > > I agree with you that it is a little bit inconvenient because the color is a little bit too transparent, does this look better to you?
> > > Dark mode:
> > > ![image](https://user-images.githubusercontent.com/76709894/128597764-6ef3bd62-70bb-40a2-ab73-ffc4eda2effa.png)
> > > Light mode:
> > > ![image](https://user-images.githubusercontent.com/76709894/128597767-a42606c5-2a80-4572-8fbf-71c1db14784c.png)
> > > Or you meant that the currently active mode icon shouldn't have a modifier color but the inactive mode icon should have it? I think the idea here is the currently active mode has that greenish color to stand out and signals that it is the current active one.
> > 
> > 
> > I can purpose this change in a PR too, btw. I personally think it fits better and more clear.
> 
> Assalam o alikum, I also noted this issues, its really confusing in terms of UX, I clicked light bulb for switching to light mode but it is disabled and currently highlighted 😕 if it's done intentionally (the highlight color) so, why the button which in not currently selected is disabled? It should toggle to light mode when I click the "light buld".

I think I understand the issue better now and I agree with you and the OP, it is indeed confusing and inconvenient... it is so confusing that I was trying to make a case for it even and defend it.
